de.dlr.sc.virsat.fdir.test/launch/VirSat FDIR All Tests.launch

```diff
@@ -53,6 +53,7 @@
 <setEntry value="de.dlr.sc.virsat.license.feature:default"/>
 <setEntry value="de.dlr.sc.virsat.model.calculation.feature:default"/>
 <setEntry value="de.dlr.sc.virsat.model.extension.fdir.feature:default"/>
+<setEntry value="de.dlr.sc.virsat.model.extension.fs.feature:default"/>

```

Comment:
Why is this file changed? I dont think you did anythng here.
Replace with development version.